src/microceph.py

```diff
@@ -232,6 +232,30 @@ def join_cluster(token: str, micro_ip: str = "", **kwargs):
     _run_cmd(cmd=cmd)
 
 
+def enable_nfs(target: str, cluster_id: str) -> None:
+    """Enable the NFS service on the target host with the given Cluster ID."""
+    cmd = ["microceph", "enable", "nfs", "--target", target, "--cluster-id", cluster_id]
+    _run_cmd(cmd)
+
+
+def disable_nfs(target: str, cluster_id: str) -> None:
+    """Disable the NFS service on the target host with the given Cluster ID."""
+    cmd = ["microceph", "disable", "nfs", "--target", target, "--cluster-id", cluster_id]
+    _run_cmd(cmd)
+
+
+def create_fs_volume(volume_name: str) -> None:
+    """Create the FS volume."""
+    cmd = ["microceph.ceph", "fs", "volume", "create", volume_name]
+    _run_cmd(cmd)
+
+
+def get_fs_volumes() -> List[dict]:
```

Comment:
should be called `list_fs_volumes`, a `get` would suggest fetching information for a single (and requested) fs volume.

Reply:
Done